src/index.js

```diff
@@ -31,7 +52,12 @@ const compileStylusFile = (jsFile, stylusFile) => {
     throw Error('Cannot find stylus file: ' + stylusFile);
   }
 
-  const stylusContent = fs.readFileSync(path, 'utf8');
+  let stylusContent = fs.readFileSync(path, 'utf8');
+
+  if (stylusContent.indexOf('~') !== -1) {
+    stylusContent = convertNodeModuleImport(stylusContent);
+  }
+
   return stylus(stylusContent)
     .include(dirname(path))
     .include(resolve('./node_modules'))
```

Comment:
Awesome catch here; the stylus render process does all of this work. I'm going to push a commit onto my feature branch which removes this unnecessary logic.

I've also changed the regex pattern I'm matching modules on. I've included some comments in the code to explain why, but the jist is that matching on `~` is too naive, since `~` is a css selector. By matching on `~` when `@require` or `@import` occurs in the same line, we can guarantee that this is a correct match.